net-p2p/bitcoin-qt/bitcoin-qt-0.21.1.ebuild

```diff
@@ -0,0 +1,206 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DB_VER="4.8"
+inherit autotools bash-completion-r1 db-use desktop xdg-utils
+
+BITCOINCORE_COMMITHASH="194b9b8792d9b0798fdb570b79fa51f1d1f5ebaf"
+KNOTS_PV="${PV}.knots20210629"
+KNOTS_P="bitcoin-${KNOTS_PV}"
+
+DESCRIPTION="An end-user Qt GUI for the Bitcoin crypto-currency"
+HOMEPAGE="https://bitcoincore.org/ https://bitcoinknots.org/"
+SRC_URI="
+	https://github.com/bitcoin/bitcoin/archive/${BITCOINCORE_COMMITHASH}.tar.gz -> bitcoin-v${PV}.tar.gz
+	https://bitcoinknots.org/files/0.21.x/${KNOTS_PV}/${KNOTS_P}.patches.txz -> ${KNOTS_P}.patches.tar.xz
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+
+IUSE="+asm +berkdb dbus kde knots +qrcode sqlite +system-leveldb test upnp +wallet zeromq"
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="
+	sqlite? ( wallet )
+	berkdb? ( wallet )
+	wallet? ( || ( berkdb sqlite ) )
+"
+RDEPEND="
+	>=dev-libs/boost-1.58.0:=[threads(+)]
+	>dev-libs/libsecp256k1-0.1_pre20200911:=[recovery,schnorr]
+	>=dev-libs/univalue-1.0.4:=
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtwidgets:5
+	system-leveldb? ( virtual/bitcoin-leveldb )
+	dbus? ( dev-qt/qtdbus:5 )
+	dev-libs/libevent:=
+	qrcode? (
+		media-gfx/qrencode:=
+	)
+	sqlite? ( >=dev-db/sqlite-3.7.17:= )
+	upnp? ( >=net-libs/miniupnpc-1.9.20150916:= )
+	berkdb? ( sys-libs/db:$(db_ver_to_slot "${DB_VER}")=[cxx] )
+	zeromq? ( net-libs/zeromq:= )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	>=sys-devel/autoconf-2.69
+	>=sys-devel/automake-1.13
+	dev-qt/linguist-tools:5
+	knots? (
+		gnome-base/librsvg
+		media-gfx/imagemagick[png]
+	)
+"
+
+DOCS=(
+	doc/bips.md
+	doc/bitcoin-conf.md
+	doc/descriptors.md
+	doc/files.md
+	doc/JSON-RPC-interface.md
+	doc/psbt.md
+	doc/reduce-traffic.md
+	doc/release-notes.md
+	doc/REST-interface.md
+	doc/tor.md
+)
+
+S="${WORKDIR}/bitcoin-${BITCOINCORE_COMMITHASH}"
+
+pkg_pretend() {
+	if use knots; then
+		elog "You are building ${PN} from Bitcoin Knots."
+		elog "For more information, see:"
+		elog "https://bitcoinknots.org/files/0.21.x/${KNOTS_PV}/${KNOTS_P}.desc.html"
+	else
+		elog "You are building ${PN} from Bitcoin Core."
+		elog "For more information, see:"
+		elog "https://bitcoincore.org/en/2021/05/01/release-${PV}/"
+	fi
+	elog
+	elog "Replace By Fee policy is now always enabled by default: Your node will"
+	elog "preferentially mine and relay transactions paying the highest fee, regardless"
+	elog "of receive order. To disable RBF, set mempoolreplacement=never in bitcoin.conf"
+	if has_version "<${CATEGORY}/${PN}-0.21.1" ; then
```

Comment:
Do we want a blank elog line here?

Reply:
IMHO, it doesn't seem necessary. The output looks fine as-is:
![image](https://user-images.githubusercontent.com/194713/125178566-62a0cb80-e1b4-11eb-8f4a-b00744e536c2.png)